_posts/2023-06-28-trino-fest-2023-starburst-recap.md

```diff
@@ -0,0 +1,42 @@
+---
+layout: post
+title: "Zero-cost reporting"
+author: "Jan Waś, Cole Bowden"
+excerpt_separator: <!--more-->
+image: /assets/blog/trino-fest-2023/Starburst.png
+show_pagenav: false
+---
+
+Let's say you have some data. Maybe it's in a spreadsheet, a CSV file, a
```

Comment:
I feel like this paragraph is a bit long to be all before the "more" break. Can we separate it up somehow?